props/saber_fett263_buttons.h

```diff
@@ -257,7 +326,102 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+#ifdef FETT263_BATTLE_MODE
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_AUX):
+         if (!battle_mode_) {
+           battle_mode_ = true;
+           hybrid_font.SB_Force();
+         } else {
+           battle_mode_ = false;
+           hybrid_font.SB_Force();
+         }
+         return true;
+
+       // Auto Lockup Mode
+       case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON):
+         if (battle_mode_) {
+           clash_impact_millis_ = millis();
+           swing_blast_ = false;
+           if (clash_impact_millis_ > last_swing_) {
+             SaberBase::SetLockup(SaberBase::LOCKUP_NORMAL);
+             swing_blast_ = false;
+             auto_lockup_on_ = true;
+             SaberBase::DoBeginLockup();
+             return true;
+	   }
+         } else {
+           return false;
+         }
+
+       case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_ON):
+         if (battle_mode_) {
```

Comment:
for simplicity (and easier indentation) it might be better to do:

if (!battle_mode_) return false;
// rest of code here